### PR TITLE
Pre translate refactor

### PR DESCRIPTION
**Pull request by gysipos**

### CODE REVIEW

File: dashboard/menu/dashboard/pretranslate.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
 # Pre-translate
 
-By using the Pre-translate menu, you can initiate a process which supplies untranslated segments with a preliminary translation in order to mitigate bleedthrough on the proxied pages. You can use pre-translate with a Translation Memory or a Machine Translation API of your choice. the translation proxy supports Google Translate, iTranslate4eu, Bing Translate and GeoFluent.
+By using the Pre-translate menu, you can initiate a process which supplies untranslated segments with a preliminary translation in order to mitigate bleedthrough on the proxied pages. You can use pre-translate with a Translation Memory as well as any of the supported Machine Translation APIs.
 
-In order to use Machine Translation, you need to go into your Account and set up your MT account Ids/authorization keys in the "Machine Translation" menu.
+In order to use Machine Translation, you need at least one API configured for your project. You can access configuration for available Machine Translators by clicking (configure) next to the desired Trasnlator, or by selecting them on in the menu bar.
```
Comment (zmeggyesi) [inline dashboard/menu/dashboard/pretranslate.md]: Consider using italics to distinguish control labels - simple parens de-emphasize content.

File: dashboard/cookbook/sitesearch.md [hunk 2]
```diff
@@ -1,34 +1,33 @@
 # Site Search
 
-Transparent multilingual search is a frequently required (though often belatedly acknowledged) part of website translation. We recommend a client-side approach to search (the approach to which the proxy translation model lends itself best), a comparatively involved use case of code injection. Given its pervasiveness and importance, it warrants detailed treatment in our documentation.
+If a site includes a search function of any kind (ex.: product search in a webshop), visitors expect to be able to search for things in the language of the website -- to input any term into the search fields and receive appropriate results immediately.
 
-**Part I** of this recipe provides a general description of the site search issue and an introduction to the recommended solution. **Part II** contains a simple example implementation.
+**A pretty standart example**
```
[the remark this answers is not in the document]
Comment (gysipos) [reply dashboard/cookbook/sitesearch.md]: fixed

File: dashboard/cookbook/sitesearch.md [hunk 3]
```diff
@@ -1,34 +1,33 @@
...
 
 There are two aspects to this kind of solution that you should consider:
 
 On the one hand, such a site search has to be coded in JavaScript in the form of an override (an example with a detailed explanation follows below), and depending on how ornate/feature-laden the original's search functionality is, complexity of the override implementation can vary from the relatively simple to the astonishingly complex.
 
-On the other hand, familiarity with the various indexing-related conditions of your chosen vendor is also important (we provide pointers to documentation for Bing, since it is the vendor used in the example). In our experience, it is not at all unusual for search engines to be leisurely in their pace. Remember that a site can only be indexed after it is published over the proxy.
+On the other hand, familiarity with the various indexing-related conditions of your chosen vendor is also important (we provide pointers to some documentation for Bing, since it is the vendor used in the example). In our experience, it is not at all unusual for search engines to act leisurely in their pace. (Remember that a site can only be indexed after it is published over the proxy.)
```
[the remark this answers is not in the document]
Comment (gysipos) [reply dashboard/cookbook/sitesearch.md]: removed

File: dashboard/cookbook/sitesearch.md [hunk 4]
```diff
@@ -1,34 +1,33 @@
...
 
-On the same note, the [publishing method](../../dashboard/index.html#publish-website) you use can also have a bearing on the way search & indexing will work, so a case-by-case analysis is a must. Note that the Client-Side Translation publishing method is *not compatible* with a site search integration of the type described here.
+On the same note, the [publishing method](../../dashboard/index.html#publish-website) you use can also have a bearing on the way search & indexing will work, so a case-by-case analysis is a must. (Note that the Client-Side Translation publishing method is *not compatible* with a site search integration of the type described here.)
```
[the remark this answers is not in the document]
Comment (gysipos) [reply dashboard/cookbook/sitesearch.md]: removed

File: dashboard/cookbook/sitesearch.md [hunk 5]
```diff
@@ -104,9 +103,9 @@ Site search functionality is frequently displayed to the visitor in the form of
 
 We use jQuery's `$(document).ready()` to wait until the DOM is ready to be manipulated. The IIFE wrapper isolates our modifier from the global namespace, and we never leave the house without `use strict`.
 
-We could define a callback on `document.onreadystatechange`, but this approach suffers from potential problems: we'd be setting our modifier on a publicly accessible property of `document`, exposing us to the possibility that some other script might inadvertently redefine it (or we could be the ones doing the redefining).
+(We could define a callback on `document.onreadystatechange`, but this approach suffers from potential problems: we'd be setting our modifier on a publicly accessible property of `document`, exposing us to the possibility that some other script might inadvertently redefine it (or we could be the ones doing the redefining).
```
[the remark this answers is not in the document]
Comment (gysipos) [reply dashboard/cookbook/sitesearch.md]: removed

File: dashboard/cookbook/sitesearch.md [hunk 6]
```diff
@@ -104,9 +103,9 @@ Site search functionality is frequently displayed to the visitor in the form of
...
 
-We would also end up having to pool different page modifiers in one function definition, which goes against the principle of separation of concerns. For these and similar reasons, vanilla JavaScript's `document.addEventListener`s or jQuery are a better fit.
+We would also end up having to pool different page modifiers in one function definition, which goes against the principle of separation of concerns. For these and similar reasons, vanilla JavaScript's `document.addEventListener`s or jQuery are a better fit.)
```
[the remark this answers is not in the document]
Comment (gysipos) [reply dashboard/cookbook/sitesearch.md]: Funny, i don't have parentheses here in my local version.

File: dashboard/menu/dashboard/pretranslate.md (hunk 1 above)
Comment (gysipos) [reply dashboard/menu/dashboard/pretranslate.md]: On the ui it's lin parenthesen, i used it as a visual guide. Understand your point, i switched to italic

File: dashboard/index.rst [hunk 7]
```diff
@@ -93,7 +93,6 @@ Cookbook
    cookbook/requestevaluation
    cookbook/stagingdomain
    cookbook/rtlconversion
-   cookbook/sitesearch
```
Comment (zmeggyesi) [inline dashboard/index.rst]: Put this back :)

File: dashboard2/index.rst [hunk 8]
```diff
@@ -11,6 +11,9 @@ The Dashboard 2.0
 .. toctree::
    publishwebsite
 
+.. toctree::
```
Comment (zmeggyesi) [inline dashboard2/index.rst]: Wrong item to be moved

File: dashboard/index.rst (hunk 7 above)
Comment (gysipos) [reply dashboard/index.rst]: i have put it back

**Pull request merged**